src/graphql/mongoose_graphql_sse_handler.erl

```diff
@@ -26,6 +26,10 @@
           {shutdown, cowboy:http_status(), cowboy:http_headers(), iodata(), req(), state()}.
 init(State, _LastEvtId, Req) ->
     process_flag(trap_exit, true), % needed for 'terminate' to be called
+    % set 1h timeout to prevent from frequent client disconections
```

Comment:
```suggestion
    % set 1h timeout to prevent from frequent client disconnections
```
I would make it configurable. Also: for me it's "prevent (sth) from (sth)"